src/PrestaShopBundle/Form/Admin/Sell/Catalog/AttributeType.php

```diff
@@ -106,17 +106,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'help' => $this->trans('Your internal name for this attribute.', 'Admin.Catalog.Help')
                     . '&nbsp;' . $this->trans('Invalid characters:', 'Admin.Notifications.Info')
                     . ' ' . TypedRegexValidator::CATALOG_CHARS,
-            ]);
-
-        if ($hasAttributeGroupId === true && $attributeGroup->group_type === AttributeGroupType::ATTRIBUTE_GROUP_TYPE_COLOR) {
```

Comment:
I added this check in my recent PR (https://github.com/PrestaShop/PrestaShop/pull/35865/files#diff-010f6eb7f9b60c2911e7db12103d0575cdacdcb29f4da24cfb67be0bbf1fd996R111). Are you sure that we can remove it?

Reply:
I think it's required if we want to toggle the fields dynamically